src/pages/aclmanagement.html

```diff
@@ -14,79 +14,162 @@ <h1 id="acl-management-view-title">
+                <table class="acl-rules table table-striped table-hover" aria-describedby="ACL rules table">
+                    <thead>
+                    <tr>
+                        <th scope="col" class="index-column">{{'acl_management.rulestable.column.index' | translate}}
+                        </th>
+                        <th scope="col" class="reorder-column"></th>
+                        <th scope="col" class="subject-column">{{'acl_management.rulestable.column.subject' | translate}}
+                        </th>
+                        <th scope="col" class="predicate-column">
+                            {{'acl_management.rulestable.column.predicate' | translate}}
+                        </th>
+                        <th scope="col" class="object-column">
+                            {{'acl_management.rulestable.column.object' | translate}}
+                        </th>
+                        <th scope="col" class="context-column">{{'acl_management.rulestable.column.context' | translate}}
+                        </th>
+                        <th scope="col" class="role-column">{{'acl_management.rulestable.column.role' | translate}}</th>
+                        <th scope="col" class="policy-column">
+                            {{'acl_management.rulestable.column.policy' | translate}}
+                        </th>
+                        <th scope="col" class="actions-column"></th>
+                    </tr>
+                    <tr>
+                        <th scope="colgroup" colspan="9" class="toolbar">
+                            <div ng-if="editedRuleIndex === undefined" class="pull-right">
+                                <button ng-click="addRule(0)" class="btn btn-link add-rule-btn"
+                                        gdb-tooltip="{{'acl_management.rulestable.actions.add_rule' | translate}}">
+                                    <em class="icon-plus"></em>
+                                </button>
+                            </div>
+                        </th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    <tr ng-hide="!loading">
+                        <td colspan="9">
+                            <div onto-loader-new size="100" style="height: 75vh; display: flex;">
+                            </div>
+                        </td>
+                    </tr>
+                    <tr ng-if="!rulesModel.size()">
+                        <td colspan="9" class="no-data">
+                            {{'acl_management.rulestable.messages.no_data' | translate}}
+                        </td>
+                    </tr>
+                    <tr ng-repeat-start="rule in rulesModel.aclRules track by $index" ng-if="0"></tr>
+                    <tr ng-if="$index !== editedRuleIndex" class="acl-rule preview-rule-row"
+                        ng-class="{'selected': $index === selectedRule || $index === editedRuleIndex}">
```

Comment:
`$index === editedRuleIndex` always be false

Reply:
I don't get this. We should discuss it. I'll eventually fix it with the next MR if needed.